src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ spectre_target_sources(
   KerrHorizon.cpp
```

Comment:
Can you change the commit message from `SphKerrSchild` to `Spherical KerrSchild` when squashing?